libs/coin-framework/src/currencies/support.ts

```diff
@@ -52,22 +52,17 @@ export const locallySupportedFiats = [
 ];
 
 async function initializeUserSupportedFiats() {
-  try {
-    const ids = await fetchSupportedFiatsTokens();
-    let supportedTokens = locallySupportedFiats;
-
-    if (ids.length) {
-      const idsToUpper = ids?.map(id => id.toUpperCase());
-
-      // This makes sure we only keep the elements supported in our API and that are available for getFiatCurrencyByTicker
-      supportedTokens = idsToUpper.filter(token => locallySupportedFiats.includes(token));
+  const ids = await fetchSupportedFiatsTokens();
+  const supportedTokens = locallySupportedFiats;
```

Comment:
We don't need this instruction. As we copy the reference when we do `supportedTokens.push(token);` we append to locallySupportedFiats too.

Reply:
Modified the function for a clearer code.